common/auth/tls.go

```diff
@@ -42,5 +42,7 @@ type (
 		EnableHostVerification bool `yaml:"enableHostVerification"`
 
 		ServerName string `yaml:"serverName"`
+
+		CaData string `yaml:"caData"` //optional inline base64 encoded ca cert
```

Comment:
nit (up to you):
```suggestion
		CAData string `yaml:"caData"` //optional inline base64 encoded ca cert
```

Reply:
I do like that, but I was trying to match the above `CaFile` - maybe come back to this at a larger scale in future.